test/api/v3/integration/tasks/DELETE-tasks_id.test.js

```diff
@@ -16,7 +16,7 @@ describe('DELETE /tasks/:id', () => {
     let task;
 
     beforeEach(() => {
-      return user.post('/tasks', {
+      return user.post('/tasks?tasksOwner=user', {
```

Comment:
I agree that a separate route would be better and cleaner. We should figure out how to make express respect that, rather than go the query parameter route.